Documentation/guides/messages/xa1026.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Xamarin.Android error/warning XA1026
+description: XA1026 error/warning code
+ms.date: 09/29/2020
+---
+# Xamarin.Android error/warning XA1026
+
+## Example messages
+
+```
+warning XA1026: Using 'aapt' is deprecated in favor of 'aapt2'. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.
+```
+
+```
+error XA1026: Using 'aapt' is not supported in Xamarin.Android projects that target .NET 5 or higher. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.
+```
+
+## Issue
+
+[Google has deprecated][aapt] the `aapt` command-line tool in favor of
+`aapt2` going forward. Xamarin.Android has now deprecated
+`<AndroidUseAapt2>false</AndroidUseAapt2>` as well.
+
+[aapt]: https://developer.android.com/studio/command-line/aapt2#aapt2_changes
+
+## Solution
+
+Update the `$(AndroidUseAapt2)` MSBuild property to `true` to select
+'aapt2'. This property corresponds to the **Use incremental Android
+packaging system (aapt2)** setting in the Visual Studio project
+properties pages. Alternatively, remove `<AndroidUseAapt2>` from the
+_.csproj_ file to let the build select the default value `true`.
```

Comment:
To match the other comment about AAPT2 naming and use the slightly more "boring" (undecorated) syntax for MSBuild properties, maybe:

```markdown
Update the `AndroidUseAapt2` MSBuild property to `true` to select
AAPT2. This property corresponds to the **Use incremental Android
packaging system (aapt2)** setting in the Visual Studio project
properties pages. Alternatively, remove `<AndroidUseAapt2>` from the
_.csproj_ file to let the build select the default value `true`.
```